src/scripts/main.js

```diff
@@ -1 +1,54 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--no-scroll');
+  } else {
+    document.body.classList.remove('page__body--no-scroll');
+  }
+});
+
+const sendButton = document.querySelector('.button--send');
+
+sendButton.addEventListener('click', formSubmit, false);
+
+function formSubmit(event) {
+  event.preventDefault();
+}
```

Comment:
currently the form is kind of irresponsive - it just does nothing.
Let's make it alive, so it at least resets and somehow react to submit.
```
const form = document.getElementById('form');
//you'll need to provide your `<form` with `id='form'`

form.addEventListener('submit', function(event) {
  event.preventDefault();
  form.reset();
});
```